src/components/textfield/TextField.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { bool, string, func,oneOf,object } from 'prop-types';
+import styles from './textfield.scss';
+import classNames from 'classnames';
+
+
+const TextField = ({value, onChange, onFocus, onBlur, label, placeholder, type="text", multiline, } )=>{
+
+   if(!multiline){
+     return(
+      <div className={classNames(styles.defaultRoot)}>
+        <label className={classNames(styles.defaultLabelStyle)}>{label}</label>
+        <input
```

Comment:
To reduce duplicate could you could do something like

```jsx
const Input = multiline ? 'textarea' : 'input';
```

then use

```jsx
<Input
```

Then you wouldln't need an `if` statement 😄

Reply:
+1